### PR TITLE
Completed Requested changes

### PR DESCRIPTION
**Pull request by asirota**
I've made the requested changes to the filenames and the name attribute in the source.

### CODE REVIEW

File: sources/wild-apricot.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
[the remark this answers is not in the document]
Comment (asirota) [reply sources/wild-apricot.json]: I have removed this file.

**Comment by asirota** [comment]
I think I've addressed every requested change.

File: sources/wild-apricot.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (asirota) [reply sources/wild-apricot.json]: I thought renaming it would eliminate the previous changes.

File: organizations/wild_apricot_personify.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "id": "WILD_APRICOT_PERSONIFY",
+    "name": "Wild Apricoty by Personify",
```
Comment (pfrisella) [inline organizations/wild_apricot_personify.json]: Should this be Apricot?

Comment (asirota) [reply organizations/wild_apricot_personify.json]: Absolutely -- I made the update. Good eye!

**Pull request merged**